tools/libfrencutils/mpp.c

```diff
@@ -130,16 +130,16 @@ void mpp_sync()
 
 void mpp_send_double(const double* data, int size, int to_pe)
 {
-#ifdef use_libMPI    
+#ifdef use_libMPI
   MPI_Status status;
   /* make sure only one message from pe->to_pe in queue */
   if(request[to_pe] != MPI_REQUEST_NULL) {
     MPI_Wait( request+to_pe, &status );
   }
-  void* temp_data = data; 
-  MPI_Isend(temp_data, size, MPI_DOUBLE, to_pe, tag, MPI_COMM_WORLD, request+to_pe);
+  /*void* temp_data = data; */
```

Comment:
Is there a reason for leaving the comment?

Reply:
No.  It was an oversite.  I'll push an update to remove the comment.